Gaming The Syllabus/Assets/Scripts/LevelLoader.cs

```diff
@@ -9,6 +9,14 @@ public class LevelLoader : MonoBehaviour
 {
```

Comment:
Consider making this class inherit from the Singleton base class. It wraps up some safety stuff to protect against accidental multiple instances. Take a look at some of the existing ones (CombatManager for example) to see how to set it up.